vizro-core/src/vizro/models/_components/figure.py

```diff
@@ -52,11 +52,9 @@ def __getitem__(self, arg_name: str):
     @_log_call
     def build(self):
         return dcc.Loading(
-            # Optimally, we would like to provide id=self.id directly here such that we can target the CSS
-            # of the children via ID as well, but the `id` doesn't seem  to be passed on to the loading component.
-            # I've raised an issue on dash here: https://github.com/plotly/dash/issues/2878
-            # In the meantime, we are adding an extra html.div here.
-            html.Div(self.__call__(), id=self.id, className="figure-container"),
```

Comment:
@huong-li-nguyen I found that [this PR](https://github.com/plotly/dash/pull/2888) is merged, but didn't succeed to move the `self.id` to the `dcc.Loading` component without messing up its appearance.

Reply:
The PR was merged on Jun 18th, but they haven't done a release since then, which is why the bug is still in there 😬 It will be officially fixed in the next Dash release. So for now, I would leave the code comments there.

Reply:
Ah, I missed that they haven't released it yet.. Thanks 😃

Reply:
Unresolving this just to remind myself of something...

Reply:
ok, all good 😀